lib/ash_rbac/policies.ex

```diff
@@ -13,18 +13,23 @@ defmodule AshRbac.Policies do
     {field_settings, action_settings} = transform_options(dsl_state)
     bypass = Info.bypass(dsl_state)
 
-    {:ok,
-     case Info.public?(dsl_state) do
-       false ->
-         dsl_state
-         |> add_field_policies(field_settings)
-         |> add_action_policies(action_settings)
-         |> add_bypass(bypass)
-
-       true ->
-         dsl_state
-         |> add_allow_policy()
-     end}
+    dsl_state =
+      cond do
+        {field_settings, action_settings} == {%{}, %{}} ->
+          dsl_state
```

Comment:
We should probably not do this here. If only bypass or public settings were set, those wouldn't be applied in this case.

we could just add another `add_field_policies` / `add_action_policies` functions that just return the dsl_state if the settings == %{}